README.md

```diff
@@ -14,52 +16,66 @@ git clone git@github.com:ONSdigital/dp-cli.git
 
 :warning: `dp-cli` uses Go Modules and **must** be cloned to a location outside of your `$GOPATH`.
 
-#### Prerequisites
+### Prerequisites
 
-`dp-cli` uses Go Modules so requires a go version of **1.11** or later.
+**Required:**
 
-`dp-cli` requires (for some functionality) the repos:
+The DP CLI uses Go Modules so requires a go version of **1.11** or later.
 
-- `dp-code-list-scripts`
-- `dp-hierarchy-builder`
+**Optional:**
 
-to be on your `$GOPATH`:
+ The following are only required for some functionality of this tool.
 
-```sh
-go get github.com/ONSdigital/dp-code-list-scripts
-go get github.com/ONSdigital/dp-hierarchy-builder
-```
+In order to use the `dp ssh` subcommand you will need:
 
-`dp-cli` also depends on `dp-setup` for  environment config:
+- [`dp-setup`](https://github.com/ONSdigital/dp-setup) cloned locally:
 
-```sh
-git clone git@github.com:ONSdigital/dp-setup.git
-```
+  ```bash
+  git clone git@github.com:ONSdigital/dp-setup
+  ```
+
+In order to use the `dp import cmd` subcommand you will need:
+
+- [`dp-code-list-scripts`](https://github.com/ONSdigital/dp-code-list-scripts) cloned locally:
+
+  ```bash
+  git clone git@github.com:ONSdigital/dp-code-list-scripts
+  ```
+
+- [`dp-hierarchy-builder`](https://github.com/ONSdigital/dp-hierarchy-builder) cloned locally:
+
+  ```bash
+  git clone git@github.com:ONSdigital/dp-hierarchy-builder
+  ```
 
 ### Configuration
 
-`dp-cli` configuration is defined in a YAML file and the CLI expects an environment variable `DP_CLI_CONFIG` with the path to that config file.
+Configuration is defined in a YAML file. By default the CLI expects the config to be in `~/.dp-cli-config.yml`. The config file location can be customised by setting `DP_CLI_CONFIG` environment variable to your chosen path.
 
-Create a new `dp-cli-config.yml` file and add the example content below (edit to match your local setup):
+The config file should contain the following (edit to match your local setup):
 
 ```yaml
```

Comment:
instead of duplicating the example, here, just put a link to the sample config file?
```suggestion
The [sample config file](./config/example_config.yml) should be tailored to suit you.

Perhaps:

`cp -i config/example_config.yml ~/.dp-cli-config.yml`

then edit (e.g. `vim ~/.dp-cli-config.yml`).
```